linux/scu/autoinstall.yaml

```
@@ -10,5 +10,11 @@ identity:
 ssh:
   install-server: true
   allow-pw: false
+apt:
+  preserve_sources_list: false
+  sources_list: |
+    deb https://us.archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse
+    deb https://us.archive.ubuntu.com/ubuntu/ jammy-updates main restricted universe multiverse
+    deb https://security.ubuntu.com/ubuntu/ jammy-security main restricted universe multiverse
```

Optional improvement: You could make this future-proof by using the `ansible_distribution_release` fact in place of the release name:

```suggestion
    deb https://us.archive.ubuntu.com/ubuntu/ {{ ansible_distribution_release }} main restricted universe multiverse
    deb https://us.archive.ubuntu.com/ubuntu/ {{ ansible_distribution_release }}-updates main restricted universe multiverse
    deb https://security.ubuntu.com/ubuntu/ {{ ansible_distribution_release }}-security main restricted universe multiverse
```

---

Unfortunately this won't work, since this file isn't used in the context of ansible, it's burned directly to the install USB.